content/docs/setup/kubernetes/multicluster-install/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Multicluster Installation
+description: Span an Istio mesh across multiple kubernetes clusters.
```

Comment:
Word choice: See my previous comment regarding the word "span" also be sure to use the correct capitalization:
description: Install an Istio mesh across multiple Kubernetes clusters.